scaffoldmaker/utils/transformation.py

```diff
@@ -0,0 +1,35 @@
+"""
+Utility functions for geometry.
+Created on Feb 14, 2019
+
+@author: Mahyar Osanlouy
+"""
+
+
+import math
+
+from scaffoldmaker.utils.vector import *
+
+def rotationMatrix(angle, direction, point=None):
```

Comment:
createRotationMatrixFromAngleAxis? There are several ways to make rotation matrices.
Also, I don't see how you can use point as you'll need a separate translation vector or use 4x4 homogeneous transformations.